packages/libs/src/sdk/api/generated/default/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-7.6.0-SNAPSHOT
+7.3.0-SNAPSHOT
```

Comment:
I think you're still getting a diff because of this change. Maybe we should exclude this file from the check

Reply:
ty <3